### PR TITLE
Update docs for beta.8 release

### PR DESCRIPTION
**Pull request by daxian-dbw**
A manual step is required for install powershell on openSUSE (same as `dotnet-sdk-2.0.0`), so the current script that automatically set up powershell on OpenSUSE doesn't work. Filed #5023 to track this.
Two scripts are affected:
- tools/download.sh
   It will continue to install the beta.6 powershell openSUSE package.
- tools/installpsh-suse.sh
   It's already broken since beta.7, because we stopped publishing suse.42.1 package in beta.7 due its end of lifecycle.

### CODE REVIEW

File: docker/release/fedora24/Dockerfile [hunk 1]
```diff
@@ -3,9 +3,9 @@ FROM fedora:24
 # TODO: Until a release of PowerShell for Fedora 24 is available,
 # this Dockerfile installs the CentOS 7 version for compatibility.
 
-ARG POWERSHELL_VERSION=6.0.0-beta.7
-ARG POWERSHELL_RELEASE=v6.0.0-beta.7
-ARG POWERSHELL_PACKAGE=powershell-6.0.0_beta.7-1.el7.x86_64.rpm
+ARG POWERSHELL_VERSION=6.0.0-beta.8
+ARG POWERSHELL_RELEASE=v6.0.0-beta.8
+ARG POWERSHELL_PACKAGE=powershell-6.0.0_beta.8-1.rhel.7.x86_64.rpm
```
Comment (adityapatwardhan) [inline docker/release/fedora24/Dockerfile]: According to Linux.md we support fedora 25 and 26. Should this file be called /fedora25/Dockerfile?

File: docker/release/fedora24/Dockerfile [hunk 2]
```diff
@@ -3,9 +3,9 @@ FROM fedora:24
...
 ARG IMAGE_NAME=microsoft/powershell:fedora24
```
Comment (adityapatwardhan) [inline docker/release/fedora24/Dockerfile]: Should this be fedora25?

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: docker/release/fedora24/Dockerfile (hunk 1 above)
Comment (daxian-dbw) [reply docker/release/fedora24/Dockerfile]: fedora 24 has reached the end of lifecycle. I will remove this file and create new docker files for 25 and 26 in a separate PR.

Comment (adityapatwardhan) [reply docker/release/fedora24/Dockerfile]: Sounds good.

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**